lib/cli/src/commands/wasmer_deserialize_module.h

```diff
@@ -10,7 +10,7 @@ extern "C" {
 extern size_t WASMER_MODULE_LENGTH asm("WASMER_MODULE_LENGTH");
 extern char WASMER_MODULE_DATA asm("WASMER_MODULE_DATA");
 
-wasm_module_t* wasmer_object_module_new(wasm_store_t* store, const char* module_name) {
+wasm_module_t* wasmer_module_new(wasm_store_t* store, const char* wasm_name) {
```

Comment:
The `wasmer_module_new` is not very descriptive. I would add back the `object` on it, since I think is more clear

Reply:
Should I also rename `wasm_module_t` to `wasm_object_module_t`?

Reply:
`wasm_module_t` is still the same right? (meaning: the returned type is stil a wasm module) So... no need to change `wasm_module_t`